forge-game/src/main/java/forge/game/CardTraitBase.java

```diff
@@ -565,8 +565,8 @@ public CardView getCardView() {
     }
 
     protected IHasSVars getSVarFallback() {
-        if (this.getKeyword() != null && this.getKeyword().getStatic() != null) {
-            return this.getKeyword().getStatic();
+        if (this.getKeyword() != null) {
+            return this.getKeyword();
```

Comment:
I've some concern here because this altering of the order could cause problems in cases like this:
```
K:Monstrosity:5:7 G G:X:creature card in your graveyard
SVar:X:Count$ValidGraveyard Creature.YouCtrl
```

Now if for some reason the host gets a different cardState (e.g. Clone) it would fail to find the X?

Reply:
I don't think so, the it's either on the same card, or use Static Ability to gain the Keyword or the Activated Ability?

Reply:
- Sacrifice _Death Kiss_ for _The Ever-Changing 'Dane_
- Activate Monstrosity
- Sacrifice some other vanilla creature
- Monstrosity resolve gives no counters 💣

Reply:
Yeah, that one should have used Static instead
Another reason why I want to change
(But I think I can revert that bit for now?)

Reply:
If no static available falling back to CardState seems safer right now

Reply:
until all effects are changed to static layers